src/sentry/runner/decorators.py

```diff
@@ -36,20 +37,31 @@ def inner(ctx, *args, **kwargs):
     return update_wrapper(inner, f)
 
 
-def log_level_option(default=None):
+def log_options(default=None):
     def decorator(f):
-        "Give ability to configure global logging level. Must be used before configuration."
+        """
+        Give ability to configure global logging level/format.
+        Must be used before configuration.
+        """
         import click
         from functools import update_wrapper
+        from sentry.logging import LoggingFormat
+        formats = [LoggingFormat.HUMAN, LoggingFormat.MACHINE]
 
         @click.pass_context
         @click.option('--loglevel', '-l', default=default,
             help='Global logging level. Use wisely.',
             envvar='SENTRY_LOG_LEVEL',
             type=CaseInsensitiveChoice(LOG_LEVELS))
-        def inner(ctx, loglevel=None, *args, **kwargs):
+        @click.option('--logformat', '-f', default=default,
```

Comment:
We probably shouldn't expose a short version of this with `-f`. This overloads what's commonly expected from a `-f` which would be a `--force`, and this is already overloaded for other commands, so we'll cause a conflict. I also just don't think it's valuable to have a short form since this will rarely be used.